Sources/SwiftHtml/Attributes/Global.swift

```diff
@@ -39,20 +39,23 @@ private extension String {
     }
 }
 
-private extension Array where Element == String {
+private extension Set where Element == String {
```

Comment:
All the class manipulation functions are routed through `Set<String>` `.union()`, `.subtraction()`, `.intersection()`... so change extension from `Array` to `Set`